clang/tools/libclang/CCAS.cpp

```diff
@@ -415,26 +416,34 @@ CXCASReplayResult clang_experimental_cas_replayCompilation(
   auto *DiagsPrinter = new TextDiagnosticPrinter(DiagOS, DiagOpts.get());
   DiagnosticsEngine Diags(DiagID, DiagOpts.get(), DiagsPrinter);
 
+  SmallVector<const char *, 256> Args(argv, argv + argc);
+  llvm::BumpPtrAllocator Alloc;
+  bool CLMode = driver::IsClangCL(
```

Comment:
Is this really necessary, the function accepts cc1 arguments, not driver ones.

Reply:
Done in latest push.